tests/software/stringtie/main.nf

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { STRINGTIE as STRINGTIE_FORWARD } from '../../../software/stringtie/main.nf'  addParams( options: [ publish_dir:'test_stringtie_forward' ] )
+include { STRINGTIE as STRINGTIE_REVERSE } from '../../../software/stringtie/main.nf'  addParams( options: [ publish_dir:'test_stringtie_reverse' ] )
+
+/*
+ * Test with forward strandedness
+ */
+workflow test_stringtie_forward {
+    def input = []
+    input = [ [ id:'test', strandedness:'forward' ], // meta map
+              [ file("${launchDir}/tests/data/bam/test.paired_end.sorted.bam", checkIfExists: true) ] ]
+
+    STRINGTIE_FOWARD (
```

Comment:
```suggestion
    STRINGTIE_FORWARD (
```

Reply:
Is that why the tests are failing?

Reply:
I'm gonna guess so :upside_down_face:

Reply:
We need to add to the CI that if the step fails then it uploads the logs so we investigate the error messages and not just guess.